features/hbx_admin/step_definitions/ui_config_steps.rb

```diff
@@ -159,3 +159,11 @@
   # USE THIS ONLY AFTER CONFIRMING THAT IT IS ABSOLUTELY NECESSARY
   sleep(1)
 end
+
+Then(/^the page is axe clean$/) do
+  # Ensures the axecore contrast tests are not run immediately after the DOM is available but prior to the page
+  # fully rendering in the browser.
+  # USE THIS ONLY AFTER CONFIRMING THAT IT IS ABSOLUTELY NECESSARY
+  sleep(1)
+  steps %(the page should be axe clean excluding "a[disabled], .disabled" according to: wcag2aa; checking only: color-contrast)
+end
```

Comment:
We have seen builds fail randomly along with a bug in trunk previously caused by a build passing with a false positive. Moving forward, we will be including the sleep in all axe clean checks using this step.